conf/hiera.yaml

```diff
@@ -0,0 +1,2 @@
+---
+# Hiera5 Global configuration file.
```

Comment:
no idea. I dug a bit through the ticket backstory, and I have to admit that I still do not have enough context to understand how this **should** work.